src/custom-elements/src/components/drawer/drawer.tsx

```diff
@@ -214,6 +229,7 @@ export class Drawer {
     this.positionValidation(this.position);
     this._animationClasses = this.active ? CLASSES.ACTIVE : '';
     this._backdropAnimationClasses = this.active ? '' : CLASSES.CLOSED;
+    document.addEventListener('click', (ev) => this._documentClickHandler(ev.target))
```

Comment:
Please, don't forget to remove event listener on componentWillUnload. If not, it will lead into a resources leak. 
You'll need to create a const with the function that will be based to addEventListener and removeEventListener. It must be exactly the same object (same reference) for both.